gen/src/ColdElecResponse.cxx

```diff
@@ -0,0 +1,53 @@
+#include "WireCellGen/ColdElecResponse.h"
```

Comment:
ColdElecRespone is a copy of ElecResponse, do we want to remove ElecResponse?

Reply:
Perhaps @brettviren can better comment on this. Or we can remove ColdElecResponse

Reply:
One or the other but not both.  I like the balance of `Cold...` vs `Warm...`.  Making them distinct will make future browsing/search of the code easier.  

To remove `ElectResponse` please `grep` all `test/`and `cfg/` for any use.

Reply:
Ok, I have implemented the dichotomy Cold-Warm everywhere in the code. For your review, you should see the updated commit in this pull request already.